src/makeXcodeproj.sh

```diff
@@ -1 +1 @@
-qmake -spec macx-xcode ../jacktrip.pro
+qmake - spec macx - xcode../ jacktrip.pro
```

Comment:
This doesn't look right. Is it clang-format on a shell script?

Reply:
It works on my Mac to remove the extra spaces:

```qmake -spec macx-xcode ../jacktrip.pro```

Reply:
But note that it generates ```jacktrip.xcodeproj``` in the src directory.